### ISSUE TITLE
Cannot update to latest version on Windows

### ISSUE THREAD

**Issue by ehrlich-b** [Nike-Inc/gimme-aws-creds#337]
I'm attempting to update to the latest version, in order to get the bugfixes for webauthn (specifically https://github.com/Nike-Inc/gimme-aws-creds/issues/284). However, I am not able to upgrade to any version higher than 2.4.0

## Expected Behavior
`pip3 install gimme-aws-creds==2.4.4` should succeed on Windows.

## Current Behavior
gimme-aws-creds==2.4.4 fails to install due to a dependency issue:
```
ERROR: Could not find a version that satisfies the requirement winrt>=1.0.20330.1; platform_system == "Windows" and python_version >= "3.7" (from ctap-keyring-device) (from versions: none)
ERROR: No matching distribution found for winrt>=1.0.20330.1; platform_system == "Windows" and python_version >= "3.7"
```

## Possible Solution
Downgrade to the version of ctap-keyring-device used in 2.4.0, which has apparently not dropped Windows support.

## Steps to Reproduce (for bugs)
1. Run `pip3 install gimme-aws-creds==2.4.4` on Windows

## Context
I am attempting to take the latest version, to get the most recent webauthn bug fixes (specifically https://github.com/Nike-Inc/gimme-aws-creds/issues/284).

## Your Environment
* Operating System and version: Windows 10 Enterprise (2004, OS Build 19041)

**Comment by markjreed** [comment]
On Windows, ctap-keyring-device depends on winrt, which does not support Python newer than 3.9. Since [it doesn't look like Microsoft is going to be updating winrt](https://stackoverflow.com/questions/69610231/why-cant-pip-find-winrt), there's now a community-maintained replacement called [winsdk](https://pypi.org/project/winsdk/).  I've submitted an [issue](https://github.com/dany74q/ctap-keyring-device/issues/7) and [PR](https://github.com/dany74q/ctap-keyring-device/pull/8/files) against c-k-d asking to swap it in for winrt. As far as I can tell that's the only fix short of switching from ctap-keyring-device to some other module or implementing its functionality locally within g-a-c.

**Comment by trent-annalect** [comment]
All of our Windows users are experiencing this issue as well.

**Comment by ayushmankd** [comment]
Any workaround for this?

**Comment by whiletrue111** [comment]
same here , is there any fix planed ? using python 3.11 on windows .

**Comment by epierce** [comment]
fixed in 2.8.2
